worker/secretrotate/secretrotate_test.go

```diff
@@ -355,16 +355,14 @@ func (s *workerSuite) TestRotateGranularity(c *gc.C) {
 		URI:             uri,
 		NextTriggerTime: now.Add(25 * time.Second),
 	}}
-	err = s.clock.WaitAdvance(time.Second, testing.LongWait, 1) // ensure some fake time has elapsed
-	c.Assert(err, jc.ErrorIsNil)
+	s.clock.Advance(time.Second) // ensure some fake time has elapsed
```

Comment:
`s.advanceClock(c, time.Second)` ?

Reply:
not here - we don't want to check for any waiting threads just advance the clock

Reply:
actually, we can make that change, here